src/vkscript/command_parser.cc

```diff
@@ -46,6 +46,9 @@ ShaderType ShaderNameToType(const std::string& name) {
   return kShaderTypeVertex;
 }
 
+// Timed execution is not supported in VkScript.
+constexpr bool kTimedExecutionSupported = false;
```

Comment:
If we default to false and move to a setter,none of these changes should be necessary